mne/commands/mne_make_scalp_surfaces.py

```diff
@@ -49,20 +49,25 @@ def run():
     parser.add_option("-n", "--no-decimate", dest="no_decimate",
                       help="Disable medium and sparse decimations "
                       "(dense only)", action='store_true')
+    parser.add_option("-i", "--allow-incomplete", dest="allow_incomplete",
+                      help="Allow the head surface to be 'incomplete' "
+                      "according to the sum of solid angles check.",
+                      action='store_true')
```

Comment:
it's not a dupe of the --force option?

Reply:
ahh yes it is. And it looks like `force` is never currently used